Gems/DiffuseProbeGrid/Code/Source/Render/DiffuseProbeGridFeatureProcessor.cpp

```diff
@@ -725,28 +741,21 @@ namespace AZ
 
             if (!diffuseProbeGridUpdatePass)
             {
-                AddPassRequest(renderPipeline.get(), "Passes/DiffuseProbeGridUpdatePassRequest.azasset", "RayTracingAccelerationStructurePass");
+                AddPassRequest(renderPipeline.get(), "Passes/DiffuseProbeGridUpdatePassRequest.azasset", "DepthPrePass");
                 AddPassRequest(renderPipeline.get(), "Passes/DiffuseProbeGridRenderPassRequest.azasset", "ForwardSubsurface");
-                AddPassRequest(renderPipeline.get(), "Passes/DiffuseProbeGridVisualizationPassRequest.azasset", "PostProcessPass");
+
+                // only add the visualization pass if there's an AuxGeom pass in the pipeline
```

Comment:
Can you explain why only add the visualization pass if there is an AuxGeom pass?

Reply:
The visualization spheres are an editor debug feature, so if the pipeline has AuxGeom functionality then it makes sense for it to have the visualization spheres as well.